test/integration_test.sh

```diff
@@ -232,18 +230,14 @@ elif [ $ss2_mode == "local" ]; then
 fi
 
 # 6. Create config.json
-# (TODO: Use Henry's script here)
```

Comment:
Comment "Create config.json" above should also change to reflect the new file name.